src/utils.rs

```diff
@@ -80,7 +80,41 @@ pub fn ident_at(root: &SyntaxNode, offset: usize) -> Option<CursorInfo> {
         }
     }?;
     let parent = ident.node().parent();
-    if let Some(attr) = parent.clone().and_then(Key::cast) {
+    if parent.clone().map(|x| x.kind() == rnix::SyntaxKind::NODE_INHERIT).unwrap_or(false) {
+        for n in parent.clone().unwrap().children_with_tokens() {
```

Comment:
If you cast `parent` to an [`Inherit`](https://docs.rs/rnix/0.8.0/rnix/types/struct.Inherit.html) you can use `.idents()` to iterate all inherited identifiers, instead of manually having to iterate the nodes and compare kinds and stuff :)

Reply:
fixed :)